### PR TITLE
[DOCFIX] Update performance tuning documentation

### PR DESCRIPTION
**Pull request by ZacBlanco**

### CODE REVIEW

**Comment by ZacBlanco** [comment]
@gpang pay extra notice to this section:

https://github.com/Alluxio/alluxio/pull/12206/files#diff-4a80ecd06f03a8fd9a1bf9396f9b4ff3R60-R65

**Comment by LuQQiu** [comment]
@gpang PTAL, thanks!

**Review by gpang — APPROVED** [review]
LGTM

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please

File: docs/en/operation/Performance-Tuning.md [hunk 1]
```diff
@@ -80,8 +86,10 @@ Also check out the [metrics system][2] for better insight in how the Alluxio ser
 
 To detect long GC pauses, Alluxio administrators can set `alluxio.master.jvm.monitor.enabled=true`
 for masters or `alluxio.worker.jvm.monitor.enabled=true` for workers.
+They are enabled by default in Alluxio 2.4.0 and newer.
```
Comment (calvinjia) [inline docs/en/operation/Performance-Tuning.md:89]: @ZacBlanco @gpang Could we include where the user can find these messages?

**Pull request merged**